web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
I agree with @henrahmagix, it's much easier and less error-prone to see the dependency lists together like

```
[
    'dep1',
    'dep2',
    function (dep1, dep2) {}
]
```

Mixing the angular and ES6 imports dynamically like this makes it quite hard to read. How about defining `API` as the whole object that is passed to `angular.service`?